lib/partition_setup.pm

```diff
@@ -236,19 +241,31 @@ sub addvg {
 
 sub addlv {
     my (%args) = @_;
-    send_key $cmd{addpart};
-    send_key 'down';
-    send_key 'down';
+
+    assert_screen 'expert-partitioner';
+    send_key $cmd{system_view};
+    send_key 'home';
+    send_key_until_needlematch('volume_management_feature', 'down');
+    send_key 'right' if is_sle('<15');
```

Comment:
Should we use `!is_storageng` or it's only sle related?

Reply:
This is sle12 related. It does not collapse list of partitions or lvm's in system view tree

Reply:
Added :)